src/main/java/com/google/firebase/auth/Tenant.java

```diff
@@ -54,6 +54,22 @@ public boolean isEmailLinkSignInEnabled() {
     return emailLinkSignInEnabled;
   }
 
+  /**
+   * Returns a new {@link UpdateRequest}, which can be used to update the attributes
+   * of this tenant.
+   *
+   * @return a non-null Tenant.UpdateRequest instance.
+   */
+  public UpdateRequest updateRequest() {
+    return UpdateRequest
```

Comment:
I think we should just set the tenantId. Other parameters will not be set, which means they won't be updated in the back-end.

Reply:
Alright this makes sense!